uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChangeTest.php

```diff
@@ -3642,6 +3642,53 @@ public function testCreateExpiryRecordsLogsExpiredBalanceChanges() {
     $this->assertEquals($expectedBroughtForwardLog, $balanceExpiryLogs[$broughtForwardExpiryRecord->id]);
   }
 
+  public function testCreateExpiryRecordsDoesTryToExpireBalanceChangesLinkedToSoftDeletedToilRequest() {
```

Comment:
I'm not sure if "Does Try To" makes a difference here, it doesn't seem to just try to, it actually does it, right?

How about

`testCreatingExpiryRecordsExpiresBalanceChangesLinkedToSoftDeletedToilRequest`?

Reply:
Should actually be `testCreateExpiryRecordsDoesNotExpireBalanceChangesLinkedToSoftDeletedToilRequest` The records linked to a soft deleted TOIl are not supposed to be expired.